src/main/java/ai/rev/speechtotext/models/asynchronous/RevAiJobOptions.java

```diff
@@ -65,7 +65,7 @@
   private Integer deleteAfterSeconds;
 
   /**
-   * Optional language parameter using ISO 639-1 2-letter language code
+   * Optional language parameter using ISO 639-1 2-letter or ISO 639-3 (3-letter) language code
```

Comment:
Since it's a documentation comment, we can add a bit more details
```suggestion
   * Optional language parameter using a supported ISO 639-1 2-letter or ISO 639-3 (3-letter) language code
   * as defined in the API reference
```